runtime/src/main/java/org/corfudb/runtime/view/stream/ReplexStreamView.java

```diff
@@ -49,6 +50,46 @@ public ReplexStreamView(final CorfuRuntime runtime,
 
     /** {@inheritDoc}
      *
+     * This is problematic in Replex since we need to translate
```

Comment:
I wonder if this is a good point to remove replex code from version 1.0? it is OS, we could get random questions on it, and will need to support. Why push this last minute code into the replex branch?